src/domains/checkout-common/99_main.tf

```diff
@@ -0,0 +1,38 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "<= 3.40.0"
+    }
+    azuread = {
+      source  = "hashicorp/azuread"
+      version = "<= 2.33.0"
```

Comment:
Same consideration as for the checkout-app domain

Reply:
See other